packages/flutter/lib/src/material/slider.dart

```diff
@@ -1818,6 +1818,7 @@ class _RenderSlider extends RenderBox with RelayoutWhenSystemFontsChangeMixin {
       final double increase = increaseValue();
       onChanged!(increase);
       onChangeEnd!(increase);
+      _state.showValueIndicator();
```

Comment:
Should we check whether _state is mounted before? Like:

```dart
    if (!_state.mounted) {
      return;
    }
```

Reply:
Agreed! Added check like code snippet in lines 1493 - 1495.